app/package.json

```diff
@@ -87,7 +87,7 @@
     "build:relay": "relay-compiler",
     "watch": "./esbuild.config.mjs dev",
     "test": "jest --config ./jest.config.js",
-    "dev": "pnpm run dev:server & pnpm run build:static && pnpm run watch",
+    "dev": "pnpm run build:static && pnpm run watch",
```

Comment:
I think you need to revert this change. You can make one called `dev:ui` if you want to run the server separately?

Reply:
addressed in c1f38bba3ed38153b8eed3f9725c2fd57f15584d